src/elements/types/RestElement.js

```diff
@@ -0,0 +1,27 @@
+import Expression from '../Expression';
+
+export default class RestElement extends Expression {
+
+    // TODO: Requires a function?
+
+    constructor(childNodes) {
+        super('RestElement', childNodes);
+    }
+
+    _acceptChildren(children) {
+        let argument = null;
+
+        children.passToken('Punctuator', '...');
+        argument = children.passNode('Identifier');
+
+        this._argument = argument;
+    }
+
+    get argument() {
+        return this._argument;
+    }
+
+    isPattern() {
```

Comment:
According to `estree` node `RestElement` is a `Pattern`: https://github.com/estree/estree/blob/master/es6.md#restelement